crates/wasm/src/shared/mod.rs

```diff
@@ -38,24 +41,26 @@ mod internal {
         components, Debuggable, Description, EntityId, Networked, Resource, Store, World,
     };
 
-    use super::{MessageType, ModuleBytecode, ModuleErrors, ModuleState, ModuleStateArgs};
+    use super::{MessageType, ModuleBytecode, ModuleErrors, ModuleState, ModuleStateMaker};
 
     components!("wasm::shared", {
         module_state: ModuleState,
         @[Store, Description["Bytecode of a WASM component; if attached, will be run."]]
         module_bytecode: ModuleBytecode,
         @[Networked, Store, Debuggable]
         module_errors: ModuleErrors,
+        @[Networked, Debuggable, Description["The ID of the module on the \"other side\" of this module, if available. (e.g. serverside module to clientside module)."]]
+        remote_paired_id: EntityId,
```

Comment:
This can be removed (they're routed to packages instead)

Reply:
Odd, this is not on a commit in my branch, I suspect it is from a commit from a merge